src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -259,15 +259,16 @@ that this verification is disruptive.
 
 #### `verify` flags
 
-| Flag                                | Description
-|:------------------------------------|:----------------------------------------------------------------------------|
-| `--connection-attempts` `<value>`   | The maximum number of connection attempts (default 2)
-| `--connection-timeout` `<value>`    | The timeout in seconds per connection attempt  (default 60)
-| `--operation-timeout` `<value>`     | Operation timeout for Kubernetes API calls (default 240)
-| `--junit-report` `<string>`         | XML report path and name (default "")
-| `--verbose`                         | Produce verbose logs during connectivity verification
-| `--only`                            | Comma separated list of specific verifications to perform
-| `--disruptive-tests`                | Enable verifications which are potentially disruptive to your deployment
+| Flag                                   | Description
+|:---------------------------------------|:----------------------------------------------------------------------------|
+| `--connection-attempts` `<value>`      | The maximum number of connection attempts (default 2)
+| `--connection-timeout` `<value>`       | The timeout in seconds per connection attempt  (default 60)
+| `--operation-timeout` `<value>`        | Operation timeout for Kubernetes API calls (default 240)
+| `--junit-report` `<string>`            | XML report path and name (default "")
+| `--verbose`                            | Produce verbose logs during connectivity verification
+| `--only`                               | Comma separated list of specific verifications to perform
+| `--disruptive-tests`                   | Enable verifications which are potentially disruptive to your deployment
+| `--image-override` `<string>=<string>` | Component image override. This flag can be used more than once (example: `--image-override=submariner=quay.io/myUser/submariner:latest`)
```

Comment:
Lets use a valid component name instead of "submariner".
JFYI, the valid component names are listed here - https://github.com/submariner-io/submariner-operator/blob/0b252dc397e7ee109512567636479fb3f23341ee/pkg/names/names.go#L27

```suggestion
| `--image-override` `<string>=<string>` | Component image override. This flag can be used more than once (example: `--image-override=submariner-gateway=quay.io/myUser/submariner-gateway:latest`)
```